test/cli/test_cli_prepare_fieldmap.py

```diff
@@ -110,23 +114,25 @@ def test_cli_prepare_fieldmap_default_output():
     os.remove(os.path.join(os.curdir, 'fieldmap.json'))
 
 
-@pytest.mark.prelude
 def test_cli_prepare_fieldmap_gaussian():
```

Comment:
The idea is to test the field mapping pipeline with as little external dependency as possible. So the specific unwrapper is not important for the test, but I have to specify skimage since prelude remains the default one.